Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm

```diff
@@ -832,65 +824,61 @@ static void registerLogHook()
 #ifdef NDEBUG
         // Don't send messages with types we don't want to log in release. Even though OS_LOG_TYPE_DEFAULT is the minimum,
         // the hook will be called for other subsystems with debug and info types.
-        if (type & (OS_LOG_TYPE_DEBUG | OS_LOG_TYPE_INFO))
```

Comment:
So now we're sending info logging in release builds?

Reply:
I will revert this in the next patch, to avoid extra logging.